server/routes.js

```diff
@@ -24,6 +24,7 @@ module.exports = function(app, express, passport) {
 
   app.use('/api/event', require('./api/event'));
   app.use('/api/observation', require('./api/observation'));
+  app.use('/api/attendees', require('./api/attendee'));
```

Comment:
Any objections to using /api/attendee (singular) for the base path just to be consistent with the other paths?